projects/README.md

```diff
@@ -2,5 +2,6 @@
 
 Add your name, a link to your research proposal—see [template.md](template.md)—and a one sentence summary of your project.
 
-- name - [research project title](template.md) - one sentence description
-- Alan Ren - [ml5.js Website Functionality Enhancement and Examples with Built-in Editor](alanren.md) - Enhance the ml5.js website and built-in code editor.
+* name - [research project title](template.md) - one sentence description
+* Tres - [Generative Audio in ml5.js](tres_proposal.md) - one sentence description
```

Comment:
This is great, thank you @trespimentel! Would you mind adding a one sentence description here for any additional context. (The title is actually quite sufficient but it still could be helpful for anyone looking at the overall list.)